packages/node-cli/messages.json

```diff
@@ -22,7 +22,7 @@
 	"CLI_INTERACTIVE_OPTION_DESCRIPTION": "run the command in interactive mode",
 	"CLI_TITLE": "SuiteCloud CLI for Node.js (NetSuite {0})",
 	"CLI_USAGE": "command [option]",
-	"CLI_SHOW_NOT_INTERACTIVE_COMMAND_MESSAGE" : "To execute this command in non-interactive, type: \"suitecloud {0}\"",
+	"CLI_SHOW_NOT_INTERACTIVE_COMMAND_MESSAGE" : "To execute this command in a non-interactive mode, type: \"suitecloud {0}\"",
```

Comment:
I would not add "a" . We have been saying Execute in interactive mode, or in non-interactive mode.+

Reply:
This is what David suggested. Can you please comment into the story?

Reply:
Ok! Didn't know. I have wrote a comment there. Thanks!